pymc/logprob/order.py

```diff
@@ -98,11 +101,20 @@ def find_measurable_max(fgraph: FunctionGraph, node: Node) -> Optional[List[Tens
     if axis != base_var_dims:
         return None
 
-    measurable_max = MeasurableMax(list(axis))
-    max_rv_node = measurable_max.make_node(base_var)
-    max_rv = max_rv_node.outputs
+    # logprob for discrete distribution
+    if isinstance(base_var.owner.op, RandomVariable) and base_var.owner.op.dtype.startswith("int"):
+        measurable_max = MeasurableMaxDiscrete(list(axis))
+        max_rv_node = measurable_max.make_node(base_var)
+        max_rv = max_rv_node.outputs
 
-    return max_rv
+        return max_rv
+    # logprob for continuous distribution
+    else:
+        measurable_max = MeasurableMax(list(axis))
```

Comment:
Should we add in the `MeasurableMax` docstring that it is for continuous max sub-graphs?